### ISSUE TITLE
Add dropdown completion for enum parameter types

### ISSUE THREAD

**Issue by fabianoliver** [Excel-DNA/IntelliSense#40]
Hey,

for future reference: A great extension of this project might be to add enum-type parameter tooltip support, similar to what Excel has for its built-in functions,

![excel_enum](https://cloud.githubusercontent.com/assets/12250141/20770819/53229fd6-b747-11e6-821d-91a08b974da6.jpg)

**Comment by Ranjith27** [comment]
Hi,
@govert 
Hi Govert - Regarding Excel UDF intellisense , i am able to get function argument intellisense and descriptions as well by using **ExcelDna.IntelliSense(.xll file)** . But, is there any way to achieve a intellisense with dropdown completion of enum values like the image attached below. Thanks in advance.
![intellisense-enumdropdown](https://user-images.githubusercontent.com/46099513/50392584-a7d78c00-0775-11e9-8880-e0d33d10b9de.jpg)

**Comment by sharpe5** [comment]
I would also love to have something like this. 

I did a crude workaround where if the parameter is entered is spelled incorrectly, it returns a suggestion. But it doesn't work very well.

**Comment by caioproiete** [comment]
@sharpe5 & @anzelika-sa Any chance you'd be interested in submitting a PR to add this feature?

**Comment by arkobro** [comment]
Is there any **progress** on this **feature**?

**Comment by govert** [comment]
@arkobro No progress on this, I'm afraid.

**Comment by Georgemichael961** [comment]
Hope you manage to achieve it someday

**Comment by FarazOnline** [comment]
No doubt! Would be a matchless feature, desperately awaited. Do inform whenever possible,

**Comment by adim1973** [comment]
Game changer, if possible! But I'm afraid it is not, without involving MS.

**Comment by Georgemichael961** [comment]
yes i agree, in think one of the most attractive feature in excel-dna is UDF, but unfortunately does not have the dropdown parameter functionOn 1 Apr 2023, at 06:28, adim1973 ***@***.***> wrote:﻿
Game changer, if possible! But I'm afraid it is not, without involving MS.
